code/modules/mob/living/emote.dm

```diff
@@ -107,9 +107,14 @@
 		message_animal_or_basic = custom_message
 	. = ..()
 	message_animal_or_basic = initial(message_animal_or_basic)
+	var/mob/living/carbon/carbon_user = user
+	if(!user.can_speak() || user.oxyloss >= 50)
+		return //stop the sound if oxyloss too high/cant speak
+	if(istype(carbon_user)) // For masks that give unique death sounds
+		if(isclothing(carbon_user.wear_mask) && ((carbon_user.wear_mask.visor_flags & UNIQUE_DEATH_SOUND) || (carbon_user.wear_mask.clothing_flags & UNIQUE_DEATH_SOUND)))
+			playsound(carbon_user, carbon_user.wear_mask.unique_death, 200, TRUE, TRUE)
+			return
```

Comment:
The `UNIQUE_DEATH_SOUND` flag feels redundant, I think we should just check `if(mask.unique_death) play_unique_death_sound()`.

Reply:
> The `UNIQUE_DEATH_SOUND` flag feels redundant, I think we should just check `if(mask.unique_death) play_unique_death_sound()`.

Yeah fair enough. Any ideas how to fix the runtime error with monkey_business though?